ansible/roles/nginx-application-contao4/files/contao4.conf

```diff
@@ -0,0 +1,13 @@
+root $baseRootDir/web;
+
+index app.php;
```

Comment:
@scuben shouln't you use app_dev.php, or is this diffrent to a standard symfony installlation?

Reply:
Yes, in dev it makes sense even if you need to setup user/password for basic auth with a console command first.